common/scripts/bump-beta-release-version.js

```diff
@@ -31,6 +31,12 @@ const bumpBetaVersion = (currentVersion) => {
     }
     const newBeta = Number.parseInt(betaVersion) + 1;
     return `${major}.${minor}.${patch}-beta.${newBeta}`
+  } else if (bumpType === 'stable') {
```

Comment:
This semantically isn't correct, a `stable` bump would be bump the version to the next stable version. Whereas what is happening is this is converting stable to the next beta release version

Reply:
would `beta-next` be a better name here?